airbyte-integrations/connectors/source-metabase/source_metabase/spec.yaml

```diff
@@ -14,6 +14,7 @@ connectionSpecification:
         URL to your metabase instance API
       examples:
         - "http://localhost:3000/api/"
+      pattern: "https://.+"
```

Comment:
I can bypass your regex for example 'hhhhttps://google.com' :)

Reply:
how about "^https://" ?

Reply:
good catch, thanks! done